### ISSUE TITLE
Japan hierarchy mistake: "Kyoto" is not on Shikoku; Kyoto is in the Kansai Region

### ISSUE THREAD

**Issue by jeffrey-friedl** [mapsme/omim#6104]
(I thought I reported this a year ago, but I can't find it now....)

In the map hierarchy for Japan, "Kyoto" is misfiled. Currently (in 7.3.3.1) Kyoto is filed within
"Shikoku".  It should be within "Kansai Region".

This is a matter of fact, not opinion.  Shikoku is an island, and Kyoto is not on that island. (https://en.wikipedia.org/wiki/Shikoku)

The exact meaning of "Kansai" has fluctuated over the last millennium, but it has always included Kyoto.

**Comment by lrabiet** [comment]
Is it MAPS.ME related ? Should it not be reported/fixed on OpenStreetMap side ?

**Comment by jeffrey-friedl** [comment]
I have the impression it's on the MAPS.me side, but I don't know the actual origin of this data.  Kyoto is not physically on (or even near) Shikoku Island, so a graphically hierarchical mistake of the kind one might find on OSM seems unlikely.  This kind of mistake smells like a configuration-file mistake.

**Comment by thany** [comment]
Talking about the "download maps" feature in the app? I get the impression those categories are hand-crafted, and therefor the problem is to be fixed by maps.me, not by OSM.

**Comment by jeffrey-friedl** [comment]
Yes, talking about the "download maps" feature in the app. I should have mentioned that important tidbit of context in my original note, sorry!

**Comment by Zverik** [comment]
Thanks for the report. Another user suggested that "Kyoto pref is Kinki area" — which one is more correct?

**Comment by jeffrey-friedl** [comment]
Hi Zverik,
In the case of this app and its current geographic hierarchy, adding Kyoto to the current "Kansai Region" section seems best.

More detail in case anyone cares...

"Kansai" is a common geographic breakdown for a wide area that is on par its siblings. It's the western counterpart to the "Kanto Region".  The current set of regions (Chubu/Chugoku/Hokkaido/Kansai/Kanto/Kyushu/Shikoku/Tohoku) is how a native Japanese would divvy up the country (though they would list the regions from north to south, not alphabetically). Japanese kids learn these in school.

"Kinki" is a common colloquial name for a smaller but less-well-defined region centering on Kyoto. Kyoto is certainly within both Kinki and Kansai, but where Kinki stops, and what to call the areas that border it, are vague. A map of Kyoto and its surroundings could be labeled "Kinki", but to the best of my knowledge, "Kinki" is never used when considering the whole country.  If I recall my son's education, Japanese kids learn "Kinki" in school when studying Kyoto's history (the capital of Japan was in Kyoto or nearby for ~1,500 years, prior to early 1600s).

**Comment by thany** [comment]
I think the best way to get this issue resolved is not to discuss naming convention, but to provide a PR. We have already seen the maps.me team is completely unable and/or unwilling to resolve the simplest of issues, and this one is no different.
